Makefile

```diff
@@ -222,6 +222,7 @@ e2e-setup: e2e-cleanup e2e-create-namespace e2e-deploy-3rd-party-crds
 
 .PHONY: e2e-cleanup
 e2e-cleanup: get-test-namespace
+	$(Q)-TEST_NAMESPACE=$(TEST_NAMESPACE) $(HACK_DIR)/remove-sbr-finalizers.sh
```

Comment:
instead of piling up here various cleanup steps, how about that we create a script, e.g. `$(HACK_DIR)/e2e-cleanup.sh` that would contain this logic, and the following step removes the namespace?